server/controllers/routes/activities.js

```diff
@@ -0,0 +1,11 @@
+const getClassActivities = require('../../database/quieres/getClassActivities');
+
+module.exports = (req, res, next) => {
+  const { subjectId, classId } = req.params;
+  getClassActivities(subjectId, classId)
+    .then((result) => result.rows)
+    .then((allActivities) => {
+      res.status(200).send({ statusCode: 200, allActivities });
```

Comment:
- use `json()` instead of `send()`
- `statusCode` is not necessary (it only copies the HTTP status code set by `status(200)`)
- rename `allActivities` -> `activities`